source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts

```diff
@@ -132,10 +132,12 @@ export class Route53ToApiGateway extends Construct {
       certificate: this.certificate
     });
 
-    // Create A Record in custom domain to route traffic to API Gateway
-    new route53.ARecord(this, 'CustomDomainAliasRecord', {
+    const arecordProps: route53.ARecordProps = {
       zone: this.hostedZone,
       target: route53.RecordTarget.fromAlias(new targets.ApiGateway(this.apiGateway))
-    });
+    };
+
+    // Create A Record in custom domain to route traffic to API Gateway
+    new route53.ARecord(this, 'CustomDomainAliasRecord', arecordProps); // NOSONAR
```

Comment:
Just for my understanding - this was a sonarqube finding that is being addressed?

Reply:
Yes, SonarQube thought this object was ignored since it is not assigned to anything, so wanted me to remove the instantiation.